python/essex-config/mkdocs.yml

```diff
@@ -0,0 +1,16 @@
+site_name: essex-config
+plugins:
+  - search
+theme:
+  name: material
+  features:
+    - content.code.copy
+    - content.code.select
+
+markdown_extensions:
```

Comment:
Do you have any need for mermaid diagrams?

Reply:
It also adds syntax highlighting to the code blocks :)